### ISSUE TITLE
Crash when improperly formatted filename template is used

### ISSUE THREAD

**Issue by georgemccabe** [dtcenter/METplus#674]
@jvigh discovered this area for improvement.
If a filename template has a closing curly brace '}' without a corresponding opening curly brace '{' before it, the script will crash.

## Describe the Enhancement ##
In the getraw function in metplus/util/config_metplus.py inside the elif character == "}" block there should be a check that in_brackets is set to True. If it is not, it should report an error and return. Logging an error in this function will not automatically increment the number of errors like self.log_error does in the wrappers, so it would be nice to add something to catch this and prevent the wrappers from running. This could potentially be done by returning None from getraw when this happens (also when count >10) and check for None in the configuration validation (validate_configuration_variables in metplus/util/met_util.py)

### Time Estimate ###
1 day

### Sub-Issues ###
Consider breaking the enhancement down into sub-issues.
- [X] *Add a checkbox for each sub-issue here.*

### Relevant Deadlines ###
None

### Funding Source ###
None

### Assignee ###
- [X] Select **engineer(s)** or **no engineer** required
- [X] Select **scientist(s)** or **no scientist** required

### Labels ###
- [X] Select **component(s)**
- [X] Select **priority**
- [X] Select **requestor(s)**

### Projects and Milestone ###
- [X] Review **projects** and select relevant **Repository** and **Organization** ones or add "alert:NEED PROJECT ASSIGNMENT" label
- [X] Select **milestone** to next major version milestone or "Future Versions"

## Define Related Issue(s) ##
Consider the impact to the other METplus components.
- [X] [METplus](https://github.com/dtcenter/METplus/issues/new/choose), [MET](https://github.com/dtcenter/MET/issues/new/choose), [METdatadb](https://github.com/dtcenter/METdatadb/issues/new/choose), [METviewer](https://github.com/dtcenter/METviewer/issues/new/choose), [METexpress](https://github.com/dtcenter/METexpress/issues/new/choose), [METcalcpy](https://github.com/dtcenter/METcalcpy/issues/new/choose), [METplotpy](https://github.com/dtcenter/METplotpy/issues/new/choose)

## Enhancement Checklist ##
See the [METplus Workflow](https://dtcenter.github.io/METplus/Contributors_Guide/github_workflow.html) for details.
- [X] Complete the issue definition above, including the **Time Estimate** and **Funding Source**.
- [ ] Fork this repository or create a branch of **develop**.
Branch name: `feature_<Issue Number>_<Description>`
- [ ] Complete the development and test your changes.
- [ ] Add/update log messages for easier debugging.
- [ ] Add/update unit tests.
- [ ] Add/update documentation.
- [ ] Push local changes to GitHub.
- [ ] Submit a pull request to merge into **develop**.
Pull request: `feature <Issue Number> <Description>`
- [ ] Define the pull request metadata, as permissions allow.
Select: **Reviewer(s)**, **Project(s)**, **Milestone**, and **Linked issues**
- [ ] Iterate until the reviewer(s) accept and merge your changes.
- [ ] Delete your fork or branch.
- [ ] Close this issue.

**Comment by georgemccabe** [comment]
Changes needed for #780 fixed this issue. Once #792 is merged, this issue can be closed as well.

**Comment by georgemccabe** [comment]
Resolved with PR #792
